### PR TITLE
PGD fault injection testing - initial

### PR DESCRIPTION
**Pull request by piano35-edb**
## What Changed?

DOCS-150 - Add PGD fault injection testing doc.

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+the availability and recovery of the cluster.
+
+## Requirement
+
+Ensure you meet the following requirements before using fault injection testing:
+
++ You have connected your BigAnimal cloud account with your Azure subscription.  See [Setting up your Azure Marketplace account](/biganimal/latest/getting_started/02_azure_market_setup/) for more information.
++ You should have permissions in your Azure subscription to view and delete VMs.
++ You have **pgd cli** installed.  See [Installing PGD CLI](/pgd/latest/cli/installing_cli/#) for more information. 
++ You have created a **pgd-cli-config.yml** file in your home directory.  See [Configuring PGD CLI](/pgd/latest/cli/configuring_cli/) for more information.
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:15]: Use `code formatting` for the file name (verbatim). See https://www.enterprisedb.com/docs/community/contributing/styleguide/#courier-aka-code-or-monospace-text

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+## Fault injection testing steps
+
+Fault injection testing consists of the following steps:
+
+1. Verifying Cluster Health
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:21]: Unless cluster health is a proper name, should not capitalize here (or in heading below).

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+2. Determining the write leader node for your cluster
+3. Deleting a write leader node from your cluster
+4. Monitoring cluster health
+&nbsp;
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:25]: What is the purpose of non-breaking spaces here?

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+&nbsp;
+#### Verifying Cluster Health
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:27]: Should be h3 below h2 (`##` -> `###`). See https://www.enterprisedb.com/docs/community/contributing/styleguide/#headings

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+pgd check-health -f pgd-cli-config.yml
+pgd verify cluster -f pgd-cli-config.yml
+pgd show-nodes -f pgd-cli-config.yml
+pgd show-raft  -f pgd-cli-config.yml
+pgd show-replslots –verbose -f pgd-cli-config.yml
+pgd show-subscriptions -f pgd-cli-config.yml
+pgd show-groups -f pgd-cli-config.yml
+```
+
+You can use **pgd help** for more information on these commands.
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:41]: Should be code-formatted as it is a verbatim command. See: https://www.enterprisedb.com/docs/community/contributing/styleguide/#courier-aka-code-or-monospace-text

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+## Deleting a write leader node from your cluster
+
+To delete a write lead node from the cluster:
+1. Log into BigAnimal.
+2. In a separate browser window, log into your Microsoft Azure subscription.
+3. In the left navigation of BigAnimal portal, choose **Clusters**. 
+4. Choose the cluster to test fault injection with and copy the string value from the URL. The string value is located after the underscore.  
+
+![Delete a write lead](images/biganimal_faultinjectiontest_1.png)
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:80]: Indent this three spaces to associate with list item no.4

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+&nbsp;
+
+5. In your Azure subscription, paste the string into the search and prefix it with **dp-** to search for the data plane.  
+From the results, choose the Kubernetes service from the Azure Region that your cluster is deployed in.
+
+![Delete a write lead 2](images/biganimal_faultinjectiontest_2.png)
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:86]: indent this 3 spaces to associate with list item no.5

File: product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+Don't delete the VMSS here or sub resources directly.
+!!!
+
+7. Browse to the Data Plane, choose Workloads, and locate the Kubernetes resources for your cluster to delete a chosen node.
+![Delete a write lead 3](images/biganimal_faultinjectiontest_3.png)
+
+&nbsp;
+### Monitoring cluster health
+
+After deleting a cluster node, you can monitor the health of the cluster using the same **pgd** commands that you used to verify cluster health.
```
Comment (josh-heyer) [inline product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx:104]: **pgd** -> PGD CLI

**Review by josh-heyer — CHANGES_REQUESTED** [review]
A few comments inline. Shouldn't need `&nbsp;` anywhere unless actively trying to avoid line breaks; LMK if you're having trouble getting something to format properly. 

Images 1 and 3 look blurry to my eyes (at full resolution) - might want to re-do those & see if you can get the key elements to be a bit more recognizeable.

**Review by josh-heyer — APPROVED** [review]

File: product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+3. Deleting a write leader node from your cluster
+4. Monitoring cluster health
+
+### Verifying Cluster Health
+
+Use the following commands to monitor your cluster health, node info, raft, replication lag, and write leads.  
+
+```sql
+pgd check-health -f pgd-cli-config.yml
+pgd verify cluster -f pgd-cli-config.yml
```
Comment (djw-m) [inline product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx:35]:
```suggestion
pgd verify-cluster -f pgd-cli-config.yml
```

**Pull request merged**